xarray/core/variable.py

```diff
@@ -2570,6 +2575,11 @@ def chunk(  # type: ignore[override]
         dask.array.from_array
         """
 
+        if is_extension_array_dtype(self):
+            raise ValueError(
+                f"{self} was found to be a Pandas ExtensionArray.  Please convert to numpy first."
+            )
+
```

Comment:
```suggestion
```

Reply:
No but perhaps it should be. Why remove it?

Reply:
Let's keep this minimal and get it merged. We can adapt later as needed

Reply:
Ok, but this then goes against what we decided else where i.e., do not convert to numpy by default and instead force the user to make that decision: https://github.com/pydata/xarray/pull/8723#discussion_r1486520395. Unless I am mistaken, raising an error here was the decision (similar to https://github.com/pydata/xarray/pull/8723#discussion_r1539454626 as well for raising an error and not forcing conversion).  

Also this errors anyway, but it's less helpful now:
```
FAILED xarray/tests/test_variable.py::TestVariable::test_pandas_cateogrical_no_chunk - TypeError: Cannot interpret 'CategoricalDtype(categories=[0, 1, 2, 3, 4, 5, 6, 7, 8, 9], ordered=False, categories_dtype=int64)' as a data type
FAILED xarray/tests/test_variable.py::TestVariableWithDask::test_pandas_cateogrical_dtype - TypeError: Cannot interpret 'CategoricalDtype(categories=[0, 1, 2, 3, 4, 5, 6, 7, 8, 9], ordered=False, categories_dtype=int64)' as a data type
```

Reply:
I will add this back for now so tests pass but please let me know what you think given the above!

Reply:
OK sounds good to me!